config/default.yml

```diff
@@ -4,6 +4,8 @@ AllCops:
     - 'bin/**'
     - 'config.ru'
     - 'tmp/**/*'
+    - 'db/schema.rb'
+    - 'db/migrate/201*'
```

Comment:
I think we _only_ want to exclude these in a Rails context. A ruby app might have a `db/schema.rb` or etc files that isn't auto generated/ needs linting.

Reply:
OK. I think that's something an individual app could deal with, as it would be unusual, but I agree it's surprising to see it in a non-Rails context. Note that doing this necessitates this change: https://github.com/alphagov/rubocop-govuk/pull/39.